tests/conftest.py

```diff
@@ -10,16 +14,20 @@
     framework_name="testing", framework_versions={"testing": "v1"}
 )
 
+if TYPE_CHECKING:
+    from _pytest.tmpdir import TempPathFactory
```

Comment:
what's the difference between this and `from pytest import TempPathFactory`?

Reply:
oh the autocompletion imports from _pytest

Reply:
![Screenshot 2022-06-05 at 17 45 00](https://user-images.githubusercontent.com/29749331/172071889-3a57c223-f3a0-48bd-a1c8-0b78e73208a1.png)